bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -512,6 +512,143 @@ type BootstrapTokenDiscovery struct {
 type FileDiscovery struct {
 	// KubeConfigPath is used to specify the actual file path or URL to the kubeconfig file from which to load cluster information
 	KubeConfigPath string `json:"kubeConfigPath"`
+
```

Comment:
Can we mention that if KubeConfig is set, this is also the location where we write the KubeConfig to?

Reply:
Yup I know. the current godoc comment of KubeConfigPath just reads mostly like we only read from there